### ISSUE TITLE
Support `export * from ''..."` syntax

### ISSUE THREAD

**Issue by entropitor** [Galooshi/import-js#523]
Import-js doesn't support re-exports such as `export * from "..."`

**Comment by trotzig** [comment]
Can you give a little more context on the setup? Based on [this test](https://github.com/Galooshi/import-js/blob/dc8a158d15df50b2f15be3461f73706958df986d/lib/__tests__/findExports-test.js#L471), we already support this scenario.

**Comment by entropitor** [comment]
Hmm, weird, I'll close until I can further pinpoint the issue. 

A possibility could be that because I'm using a mono-repo, import-js is just slow at updating the cache and this is why it's not working

**Comment by entropitor** [comment]
Yes, that was indeed the problem. I thought I installed it before but apparently, I didn't
